src/System.CommandLine/Symbol.cs

```diff
@@ -68,15 +68,10 @@ public virtual string Name
             }
         }
 
-        public Command Parent { get; private set; }
+        public Command Parent { get; protected internal set; }
```

Comment:
Do we really want to expose the setter for this outside of core? Is there a use case for needing it yet?